apps/src/code-studio/pd/application/teacher/ProfessionalLearningProgramRequirements.jsx

```diff
@@ -24,6 +24,10 @@ import {FormContext} from '../../form_components_func/FormComponent';
 const ProfessionalLearningProgramRequirements = props => {
   const {data} = props;
   const [regionalPartner, regionalPartnerError] = useRegionalPartner(data);
+  const hasNoProgramSelected = () => data.program === undefined;
+  const hasNoSchoolInformation = () => !data.school;
+  const hasNotLoadedRegionalPartner = () => regionalPartner === undefined;
+  const hasNoRegionalPartner = () => regionalPartner === null;
```

Comment:
i like having these defined up here, but i don't think they need to be functions, they could just be the values

Reply:
Ah yes, good call!